mmv1/templates/terraform/examples/resource_policy_consistency_group.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_compute_resource_policy" "cgroup" {
+  provider = google-beta
+
+  name   = "<%= ctx[:vars]['name'] %>"
+  region = "us-central1"
+  disk_consistency_group_policy {
+    enabled = true
```

Comment:
Can you add a handwritten test where we update from true --> false? We often have difficulties handling this transition for fields that default to true

Reply:
Absolutely. i suppose that has to be a handwritten one, or is there an easier way?

Reply:
It has to be handwritten at this point

Reply:
I think there might be an API issue here. A resource policy has to contain SOMETHING (so I can't just make `enabled = false` on a simple policy only containing that, because that would result in an empty resource policy). But I tried adding another one of the resource policies alongside it and the result was:

```
Error: Error creating ResourcePolicy: googleapi: Error 400: Invalid value at 'resource' (oneof), oneof field 'policy' is already set. Cannot set 'groupPlacementPolicy'
```

It seems wrong to me, it shouldn't be `oneof` but a combination of multiple things. But I can say the update function definitely works as otherwise I wouldn't be getting the error.

Reply:
Yes, I had a bit more look into this. You can have multiple resource policies attached (eg. one for consistency groups, one for snapshot policy) to an instance using `google_compute_disk_resource_policy_attachment` (or `gcloud compute instances add-resource-policies inst1 --resource-policies=pol1,pol2,...`). 

You can't combine multiple different types of policies in one resource policy and an empty policy is not valid, so the update tests doesn't make sense for this use case. What do you think?

Reply:
If the field can't be updated should it not then be marked as immutable ?

Reply:
That's absolutely true. Did that now and also fixed the `conflicts` parts.